content/en/css.md

```diff
@@ -29,7 +29,7 @@ Try to segment code in logical ways:
 
 Use the `<link>` tag to include all your stylesheets in the `<head>` of the document. For optimal page performance, concatenate your CSS into as few files as possible and do not use the `@import` command to include other stylesheets, as this will fire an additional HTTP request and block page rendering until its completion.
 
-```markup
+```
```

Comment:
Prism needs "markup" or "css" next to the backticks so the correct CSS class is put on the code block, so the syntax highlighting will be appropriate to the language.